sk0-back/src/constants.ts

```diff
@@ -1 +1,2 @@
 export const DATA_SOURCE = 'DataSource';
+export const USER_REPOSITORY = 'UserRepository';
```

Comment:
We might want to use `as const` for constant value declarations.

```suggestion
export const DATA_SOURCE = 'DataSource' as const;
export const USER_REPOSITORY = 'UserRepository' as const;
```

or we can use Symbols rather.

Reply:
using as const.